ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/vercel/configure/[id]/page.tsx

```diff
@@ -0,0 +1,203 @@
+  return (
+    <div className="mx-auto mt-8 flex w-[800px] flex-col p-8">
+      {fetching ? null : !project ? (
+        <Alert severity="error">Vercel project not found!</Alert>
+      ) : (
+        <div className="flex flex-col">
+          <div className="mb-2 text-2xl font-medium text-gray-900">{project.name}</div>
+          {project.ssoProtection?.deploymentType ===
+            VercelDeploymentProtection.ProdDeploymentURLsAndAllPreviews && (
+            <div className="mb-7 flex flex-row items-center justify-start text-sm font-normal leading-snug text-amber-700">
+              <RiInformationLine className="mr-2 h-4 w-4 text-amber-500" />
+              Vercel Deployment Protection might block syncing. Use the deployment protection key
+              option below to bypass.
+            </div>
+          )}
+
+          <div className="flex flex-col gap-2 rounded-lg border border-slate-200 p-6">
+            <div className="text-lg font-medium text-gray-900">Project status</div>
+            <div className="text-base font-normal text-slate-500">
+              This determines whether or not Inngest will communicate with your Vercel application.
+            </div>
+            <SwitchWrapper>
+              <Switch
+                checked={project.isEnabled}
+                className="data-[state=checked]:bg-green-600"
+                onCheckedChange={(checked) =>
+                  setProject({ ...project, isEnabled: checked, updated: true })
+                }
+              />
+              <SwitchLabel htmlFor="override" className="text-sm font-normal text-slate-500 ">
+                {project.isEnabled ? 'Enabled' : 'Disabled'}
+              </SwitchLabel>
+            </SwitchWrapper>
+          </div>
+
+          <div className="mt-4 flex flex-col gap-2 rounded-lg border border-slate-200 p-6">
+            <div className="text-lg font-medium text-gray-900">Path information</div>
+            <div className="text-base font-normal text-slate-500">
+              Each Vercel project can serve one or more Inngest apps available on different URL
+              paths.
+            </div>
+            {paths.map((path, i) => (
+              <div key={`serve-path-${i}`} className="flex flex-row items-center justify-start">
+                <div className="mr-2 w-full">
+                  <Input
+                    defaultValue={path}
+                    className="h-10 w-full px-2 py-2 text-base text-gray-900"
+                    onChange={({ target: { value } }) => {
+                      setPaths(paths.map((p, n) => (i === n ? value : p)));
+                      setProject({ ...project, updated: true });
+                    }}
+                  />
+                </div>
+
+                {paths.length > 1 && (
+                  <Button
+                    kind="danger"
+                    appearance="outlined"
+                    icon={<RiDeleteBinLine className="h-5 w-5" />}
+                    iconSide="left"
```

Comment:
nitpick: This one should probably not have an iconSide since it has no text, so it aligns to the center

Reply:
👍